.github/workflows/lint-and-test.yaml

```diff
@@ -3,49 +3,130 @@ name: Lint and test PR
 on: pull_request
 
 jobs:
-  need-test:
+  docker-build:
+    name: Build Docker image
     runs-on: ubuntu-latest
-    outputs:
-      charts: ${{ steps.list-changed.outputs.changed }}
     steps:
       - uses: actions/checkout@v3
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      - name: Build
+        uses: docker/build-push-action@v4
         with:
-          fetch-depth: 0
-      - uses: helm/chart-testing-action@v2.3.1
-      - id: list-changed
-        run: |
-          changed=$(ct --config .github/ct.yaml list-changed)
-          if [[ -n "$changed" ]]; then
-            echo "changed=true" >> $GITHUB_OUTPUT
-          fi
+          tags: ci.local/${{ github.repository }}:ci
+          cache-from: type=gha
+          cache-to: type=gha,mode=max
 
-  lint-chart:
+  chart-lint:
+    name: Lint Helm charts
     runs-on: ubuntu-latest
-    needs: need-test
-    if: ${{ needs.need-test.outputs.charts == 'true' }}
     steps:
-      - uses: actions/checkout@v3
+      - name: Checkout code
+        uses: actions/checkout@v3
         with:
           fetch-depth: 0
-      - uses: helm/chart-testing-action@v2.3.1
-      - run: helm plugin install https://github.com/helm-unittest/helm-unittest
-      - uses: manusa/actions-setup-minikube@v2.7.2
-        with:
-          minikube version: v1.29.0
-          kubernetes version: v1.25.0
-          github token: ${{ secrets.GITHUB_TOKEN }}
+      - name: Install helm/chart-testing
+        uses: helm/chart-testing-action@v2.3.1
+      - name: Lint charts
+        run: ct --config .github/ct.yaml lint --all
 
-      - run: ct --config .github/ct.yaml lint --debug
+  chart-unittest:
+    name: Unit test Helm charts
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Install helm unit test siute
```

Comment:
typo

Reply:
WDYM typo, you don't know what a test siute is??